src/components/Button.tsx

```diff
@@ -25,7 +25,7 @@ const overrideHoverEffect = css`
 const ButtonBase = styled(Button)`
 	font-family: ${props => props.theme.titleFont};
 	font-weight: ${props => normal(props)};
-	height: ${props => px(props.theme.space[4])};
+	height: 38px;
```

Comment:
Wouldn't it be nice if we can configure this through theming? I am OK with fixed value for now as well, just a thought.

Reply:
```js
  button: {
      border: {
          width: string;
          radius: string;
      };
      primary: {};
      disabled: {
          opacity: number;
      };
      minWidth: string;
      maxWidth: string;
      padding: {
          vertical: string;
          horizontal: string;
      };
  };
```

Grommet doesn't give the option in the theme.

Reply:
Yeah, I meant we can extend the theme definition, but ignore it for now.

Reply:
@dimitrisnl @sradevski Lets fix this now, and extend the theme definition with a `height` value

Reply:
@dimitrisnl can you please add `height` to the theme in this PR? Cheers

Reply:
Ah missed this. Will do 👍